lib/merchant.rb

```diff
@@ -25,7 +25,25 @@ def invoices
     repository.find_invoices(id)
   end
 
-  def revenue
+  # def revenue
+  #   merchant_transactions = invoices.map do |invoice|
+  #     invoice.transactions
+  #   end.flatten
+  #   successful_transactions = merchant_transactions.select do |transaction|
+  #       transaction.successful?
+  #   end
+  #   successful_invoices = successful_transactions.map do |transaction|
+  #     transaction.invoice
+  #   end
+  #   revenue = successful_invoices.map do |invoice|
+  #     invoice.invoice_items.reduce(0) do |sum, inv_item|
+  #       sum += inv_item.revenue
+  #     end
+  #   end.inject(:+)
+  #   revenue.to_digits
+  # end
+
+  def revenue(date=nil)
```

Comment:
Assignment Branch Condition size for revenue is too high. [22.02/15]<br>Surrounding space missing in default value assignment.